package.json

```diff
@@ -29,30 +29,29 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@dvsa/cvs-type-definitions": "7.1.0",
     "@aws-sdk/client-lambda": "^3.549.0",
     "@aws-sdk/client-s3": "^3.550.0",
     "@aws-sdk/client-secrets-manager": "^3.549.0",
-    "@aws-sdk/util-dynamodb": "^3.592.0",
+    "@aws-sdk/util-dynamodb": "^3.632.0",
+    "@dvsa/cvs-type-definitions": "7.1.0",
     "@smithy/util-utf8": "^2.3.0",
     "aws-lambda": "^1.0.7",
     "aws-xray-sdk": "^3.6.0",
     "js-yaml": "^3.13.1",
     "moment": "^2.29.4",
-    "moment-timezone": "^0.5.27",
     "node-yaml": "^4.0.1",
     "notifications-node-client": "^7.0.4",
     "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
-    "@commitlint/cli": "^13.1.0",
+    "@commitlint/cli": "^19.4.0",
     "@commitlint/config-conventional": "^13.1.0",
     "@types/aws-lambda": "^8.10.36",
     "@types/chai": "^4.2.5",
     "@types/jest": "^25.2.3",
     "@types/jest-plugin-context": "^2.9.2",
     "@types/js-yaml": "^3.12.1",
-    "@types/moment-timezone": "^0.5.12",
+    "moment-timezone": "^0.5.45",
```

Comment:
This code is falling over in the lambda, I believe the @types has accidentally been removed.